tests/src/test/java/org/jboss/forge/furnace/container/guice/service/AddonRegistryTest.java

```diff
@@ -57,4 +61,18 @@ public void testServiceInjection() throws Exception
       Assert.assertThat(service.isUnsatisfied(), is(false));
       Assert.assertThat(service.get(), notNullValue());
    }
+
+   @Test
+   public void testExportedTypes() throws Exception
```

Comment:
run this test aside of other changes to see the reason why I did this change. A lot of stuff is exported